CONTRIBUTION.md

```diff
@@ -0,0 +1,23 @@
+1. **Familiarize with the RFC Template**: Before submitting an RFC, please review our [RFC Template](template.md). This template provides a structured format for proposing changes.
```

Comment:
This is copy/paste from the existing README.md - just thought it was cleaner to move here. Open to keeping in the main file though